system-test/spanner.ts

```diff
@@ -45,15 +45,10 @@ describe('Spanner', () => {
     },
   };
 
-  before(done => {
-    async.series(
-        [
-          deleteTestResources,
-          (next) => {
-            instance.create(INSTANCE_CONFIG, execAfterOperationComplete(next));
-          },
-        ],
-        done);
+  before(async () => {
+    await deleteTestResources;
+    await instance.create(INSTANCE_CONFIG);
+    await execAfterOperationComplete;
```

Comment:
That doesn't look right :) What is `execAfterOperationComplete`?  Is that a promise?

Reply:
It's not a Promise, should not call it this way. 
I'll check if I can promisify this and call then